script/scan-prereqs-cpanfile

```diff
@@ -74,6 +76,14 @@ if ($version) {
         $develop_prereqs = scan_test_requires($dir, $develop_prereqs);
     }
 
+    # Get minimum versions from currently installed modules (= available in @INC)
+    if ($match_versions) {
+		require V;
+        for my $prereqs ($runtime_prereqs, $configure_prereqs, $test_prereqs, $develop_prereqs) {
+            $prereqs = { map { $_ => V::get_version($_) } keys %$prereqs };
```

Comment:
Yeah, sorry, I didn't look hard enough for a proper module.
But then I'd rather would like to use Module::Metadata which is a core module, any concerns about this?